lib/readline.js

```diff
@@ -429,16 +428,15 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
-        // If there is a common prefix to all matches, then apply that
-        // portion.
-        var f = completions.filter(function(e) { if (e) return e; });
-        var prefix = commonPrefix(f);
-        if (prefix.length > completeOn.length) {
-          self._insertString(prefix.slice(completeOn.length));
-        }
-
+      // If there is a common prefix to all matches, then apply that portion.
+      const f = completions.filter(function(e) { if (e) return e; });
```

Comment:
@addaleax 
we don't need to run this block of code when `completions.length === 1`, right?